<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>751</width>
    <height>518</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Parametrized  Snake Gaits</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>0</y>
      <width>441</width>
      <height>481</height>
     </rect>
    </property>
    <property name="title">
     <string>Predesigned Gaits</string>
    </property>
    <widget class="QPushButton" name="pushButton_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>121</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Side-
winding</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../sw.jpg</normaloff>../sw.jpg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_9">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>320</y>
       <width>121</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Helix
Rolling</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../he.jpg</normaloff>../he.jpg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_10">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>410</y>
       <width>121</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Lateral
Rolling</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../ro.jpg</normaloff>../ro.jpg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_11">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>320</y>
       <width>121</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Linear
Prog.
HighAmp.</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../lpha.jpg</normaloff>../lpha.jpg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_12">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>220</y>
       <width>121</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Linear
Prog.
Offset</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../lpo.jpg</normaloff>../lpo.jpg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_13">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>120</y>
       <width>121</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Linear
Prog.</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../lp.jpg</normaloff>../lp.jpg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_14">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>40</y>
       <width>121</width>
       <height>61</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>Home</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../h.jpg</normaloff>../h.jpg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_15">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>120</y>
       <width>121</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Side-
Winding
LowAmp.</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../swl.jpg</normaloff>../swl.jpg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
    </widget>
    <widget class="QCheckBox" name="torque">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>400</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>DAQ Torque</string>
     </property>
     <property name="tristate">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="velocidad">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>420</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>DAQ Speed</string>
     </property>
     <property name="tristate">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>81</width>
      <height>481</height>
     </rect>
    </property>
    <property name="title">
     <string>Joints</string>
    </property>
    <widget class="QWidget" name="layoutWidget_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>61</width>
       <height>441</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QCheckBox" name="q1">
        <property name="text">
         <string>Q1</string>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="q2">
        <property name="text">
         <string>Q2</string>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="q3">
        <property name="text">
         <string>Q3</string>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="q4">
        <property name="text">
         <string>Q4</string>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="q5">
        <property name="text">
         <string>Q5</string>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="q6">
        <property name="text">
         <string>Q6</string>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="q7">
        <property name="text">
         <string>Q7</string>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="q8">
        <property name="text">
         <string>Q8</string>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="q9">
        <property name="text">
         <string>Q9</string>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="q10">
        <property name="text">
         <string>Q10</string>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="q11">
        <property name="text">
         <string>Q11</string>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="q12">
        <property name="text">
         <string>Q12</string>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="q13">
        <property name="text">
         <string>Q13</string>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="q14">
        <property name="text">
         <string>Q14</string>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="q15">
        <property name="text">
         <string>Q15</string>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="q16">
        <property name="text">
         <string>Q16</string>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>230</y>
      <width>191</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string>Execution</string>
    </property>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>171</width>
       <height>101</height>
      </rect>
     </property>
     <property name="text">
      <string>Move</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../snake.jpg</normaloff>../snake.jpg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>120</width>
       <height>70</height>
      </size>
     </property>
     <property name="shortcut">
      <string>Ctrl+R</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>105</width>
       <height>25</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_17">
        <property name="text">
         <string>Time (s)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="tiempo">
        <property name="maximum">
         <number>100</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>200</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>About</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>0</y>
      <width>191</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>Gait Parameters</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>71</width>
       <height>191</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QSpinBox" name="offset_par">
        <property name="minimum">
         <number>-120</number>
        </property>
        <property name="maximum">
         <number>120</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="amplitud_par">
        <property name="minimum">
         <number>-120</number>
        </property>
        <property name="maximum">
         <number>120</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="offset_impar">
        <property name="minimum">
         <number>-120</number>
        </property>
        <property name="maximum">
         <number>120</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="amplitud_impar">
        <property name="minimum">
         <number>-120</number>
        </property>
        <property name="maximum">
         <number>120</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="desfase">
        <property name="minimum">
         <number>-120</number>
        </property>
        <property name="maximum">
         <number>120</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="dtheta_dn">
        <property name="minimum">
         <number>-120</number>
        </property>
        <property name="maximum">
         <number>120</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="dtheta_dt">
        <property name="minimum">
         <number>-120</number>
        </property>
        <property name="maximum">
         <number>120</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>71</width>
       <height>191</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Offset Even</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Amp. Even</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Offset Odd</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Amp. Odd</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Delta</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>dTheta/dn</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>dTheta/dt</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>751</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_14</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>home()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>355</x>
     <y>91</y>
    </hint>
    <hint type="destinationlabel">
     <x>541</x>
     <y>32</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_13</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>linear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>476</x>
     <y>163</y>
    </hint>
    <hint type="destinationlabel">
     <x>543</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_12</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>linear_offset()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>517</x>
     <y>282</y>
    </hint>
    <hint type="destinationlabel">
     <x>545</x>
     <y>310</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_11</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>linear_high()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>484</x>
     <y>382</y>
    </hint>
    <hint type="destinationlabel">
     <x>547</x>
     <y>426</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_10</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>rolling()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>380</x>
     <y>467</y>
    </hint>
    <hint type="destinationlabel">
     <x>549</x>
     <y>482</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_9</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>helix_rolling()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>189</x>
     <y>374</y>
    </hint>
    <hint type="destinationlabel">
     <x>543</x>
     <y>400</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_8</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>sidewinding()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>208</x>
     <y>274</y>
    </hint>
    <hint type="destinationlabel">
     <x>545</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_15</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>sidewinding_low()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>254</x>
     <y>185</y>
    </hint>
    <hint type="destinationlabel">
     <x>547</x>
     <y>236</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>abrir_about()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>654</x>
     <y>478</y>
    </hint>
    <hint type="destinationlabel">
     <x>723</x>
     <y>495</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>mover()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>714</x>
     <y>376</y>
    </hint>
    <hint type="destinationlabel">
     <x>615</x>
     <y>492</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>abrir_about()</slot>
  <slot>helix_rolling()</slot>
  <slot>home()</slot>
  <slot>linear()</slot>
  <slot>linear_high()</slot>
  <slot>linear_offset()</slot>
  <slot>mover()</slot>
  <slot>rolling()</slot>
  <slot>sidewinding()</slot>
  <slot>sidewinding_low()</slot>
 </slots>
</ui>
